server/node.go

```diff
@@ -517,6 +522,32 @@ func (n *Node) waitForScanCompletion() int64 {
 	return n.scanCount
 }
 
+// startLogStats periodically outputs runtime statistics to the log files.
+func (n *Node) startRuntimeStats(stopper *util.Stopper) {
+	stopper.RunWorker(func() {
+		ticker := time.NewTicker(runtimeStatsInterval)
+		defer ticker.Stop()
+		var cgoCall int64
+		ms := runtime.MemStats{}
+		mb := 10E-6
+		for {
+			select {
+			case <-stopper.ShouldStop():
+				return
+			case <-ticker.C:
+				runtime.ReadMemStats(&ms)
+				newCGOCall := runtime.NumCgoCall() - cgoCall
+				cgoRate := newCGOCall / (int64(runtimeStatsInterval) / int64(time.Second))
```

Comment:
I'm not sure how accurate the tickers are. It might be better to keep track of the time between calls explicitly.